manim/utils/bezier.py

```diff
@@ -142,7 +143,14 @@ def split_quadratic_bezier(points: Iterable[float], t: float) -> np.ndarray:
     s2 = interpolate(h1, a2, t)
     p = interpolate(s1, s2, t)
 
-    return np.array([a1, s1, p, p, s2, a2])
```

Comment:
this really makes a difference in performance?

Reply:
Well, this line not that much, but it helps because we're not creating a list for no reason and you can't create numpy arrays from generators anymore so yeah this saves one whole instantiation of a list and saves about 1 second overall with a small render. This only applies for OpenGL obviously xD But with using vprof for a while i figured that does enough that it's worth changing.

Reply:
Just because I think blowing this up to several lines is a bit silly: how about using https://numpy.org/doc/stable/reference/generated/numpy.fromiter.html#numpy-fromiter with an explicitly set dtype of float64?